config/config.go

```diff
@@ -98,14 +101,24 @@ func ReadFromFile(path string) (*Config, error) {
 		return nil, err
 	}
 
-	cfg := &Config{}
+	cfg := &Config{path: path}
 	if err = fill.Struct(fields, cfg); err != nil {
 		return nil, err
 	}
 
 	return cfg, nil
 }
 
+// SetPath assigns unexported filepath to write config to
+func (cfg *Config) SetPath(path string) {
+	cfg.path = path
+}
+
+// Path gives the unexported filepath for a config
+func (cfg Config) Path() string {
+	return cfg.path
+}
+
 // WriteToFile encodes a configration to YAML and writes it to path
 func (cfg Config) WriteToFile(path string) error {
```

Comment:
Can this change to using cfg.path instead of a parameter?

Reply:
what if I want to write config somewhere else? Do I have to `SetPath` then `WriteToFile`? I'm ok(ish) with that, just want to be sure that's the API we're after